### ISSUE TITLE
ValueError: Could not deserialize key data.

### ISSUE THREAD

**Issue by marcel-st** [benkonrath/transip-api#53]
Hi there,

i'm trying to get this app running on a Raspberry pi 3B with CentOS 7.
I've installed Python3.6 and all libraries needed and did 

```
# python3.6 setup.py build 
# python3.6 setup.py install

```
It installed without errors, but when i try to change an DNS record it fails

`# transip-api -l <username> --api-key /opt/transip-api/decrypted_key --domain-name mydomain.com --entry-name myhost --entry-expire 300 --entry-type A --entry-content 127.0.0.1
`
I get the following error :

```
Traceback (most recent call last):
  File "/usr/local/bin/transip-api", line 11, in <module>
    load_entry_point('transip==2.0.0', 'console_scripts', 'transip-api')()
  File "/usr/local/lib/python3.6/site-packages/transip-2.0.0-py3.6.egg/transip/transip_cli.py", line 122, in main
    names = domain_service.get_domain_names()
  File "/usr/local/lib/python3.6/site-packages/transip-2.0.0-py3.6.egg/transip/service/domain.py", line 48, in get_domain_names
    return self._simple_request('getDomainNames')
  File "/usr/local/lib/python3.6/site-packages/transip-2.0.0-py3.6.egg/transip/client.py", line 199, in _simple_request
    cookie = self.build_cookie(mode=kwargs.get('mode', MODE_RO), method=method, parameters=args)
  File "/usr/local/lib/python3.6/site-packages/transip-2.0.0-py3.6.egg/transip/client.py", line 182, in build_cookie
    signature = self._sign(message_to_sign)
  File "/usr/local/lib/python3.6/site-packages/transip-2.0.0-py3.6.egg/transip/client.py", line 105, in _sign
    backend=default_backend()
  File "/usr/local/lib/python3.6/site-packages/cryptography-2.8-py3.6-linux-armv7l.egg/cryptography/hazmat/primitives/serialization/base.py", line 16, in load_pem_private_key
    return backend.load_pem_private_key(data, password)
  File "/usr/local/lib/python3.6/site-packages/cryptography-2.8-py3.6-linux-armv7l.egg/cryptography/hazmat/backends/openssl/backend.py", line 1089, in load_pem_private_key
    password,
  File "/usr/local/lib/python3.6/site-packages/cryptography-2.8-py3.6-linux-armv7l.egg/cryptography/hazmat/backends/openssl/backend.py", line 1315, in _load_key
    self._handle_key_loading_error()
  File "/usr/local/lib/python3.6/site-packages/cryptography-2.8-py3.6-linux-armv7l.egg/cryptography/hazmat/backends/openssl/backend.py", line 1373, in _handle_key_loading_error
    raise ValueError("Could not deserialize key data.")
ValueError: Could not deserialize key data.
```

**Comment by mathijswesterhof** [comment]
Does your key-file have an `ssh-rsa AAAA... user@host` or does it start with `-----BEGIN PRIVATE KEY-----` ? it should be the later. I personally have it in a file With the enters and that works
I hope this might help

**Comment by benkonrath** [comment]
This has been fixed by #55 which was just merged into master.

**Comment by benkonrath** [comment]
I just released 2.1.2 which has the fix for this.
